agent/agent.go

```diff
@@ -155,6 +167,9 @@ func (a *Agent) Disconnect(remote network.ClusterAgent) error {
 }
 
 func (a *Agent) Gossip(data []byte) error {
+	a.Logger.Info("message sent",
```

Comment:
We should log message or unique part of it in order to create statistics later. I think each published message  should contain some message id (4 or 8 bytes) and rest of the message should be randomly populated as it is right now.
Only unique part of the message should be included in log because otherwise file size will be very big for large message sizes